### ISSUE TITLE
How to disable connection pool in php7?

### ISSUE THREAD

**Issue by udoless** [mongodb/mongo-php-driver#393]
When I use MongoDB\Driver mongo always creating a lot of connections and staying for a long time. How to disable this feature? I just want a short time connection. Thanks.

**Comment by udoless** [comment]
Help!

**Comment by WLWL** [comment]
I also encountered this problem,Can't see any way to close the connection.Each php-fpm process is a connection.

**Comment by derickr** [comment]
The driver is designed to leave connections open, and there is no method of turning that off. As you often don't have more than 500 PHP-FPM processes on a machine, you should not (easily) run into running out of connections to MongoDB.

**Comment by udoless** [comment]
But we have lots of php servers(more than 50), and them use the same mongodb. In one server , php-fpm processes are less than 500(about 200). As a result, mongodb's port run out easily.

**Comment by WLWL** [comment]
Our project has a similar problem.more than 4000 PHP-FPM processes.Mongo memory consumption is very large

**Comment by WLWL** [comment]
Mongoclient has the method of closing the connection but does not support php7

**Comment by braveSky** [comment]
Is there any way  can close the mongodb connection created ?

**Comment by WLWL** [comment]
Is there any other branch version mongodb to support close connection ?

**Comment by jmikola** [comment]
I've created [PHPC-780](https://jira.mongodb.org/browse/PHPC-780) to track the task of documenting the connection handling in each version of the current PHP and HHVM drivers. Part of that will be discussing how connections are persisted and including explanations of why we did not implement the same API as the legacy driver.

That said, the legacy driver's [`MongoClient::close()`](http://php.net/manual/en/mongoclient.close.php) method was deprecated well before we started developing the new driver.

In an application deployment where you're exceeding connection counts on the server (or coming close), such as 4000 FPM workers, closing connections from PHP is unlikely to resolve anything as the same connections are going to be re-established when the next request constructs a new Manager instance.
